contracts/libraries/PostLib.sol

```diff
@@ -986,6 +988,49 @@ library PostLib  {
         emit SetDocumentationTree(userAddr, communityId);
     }
 
+    function editDocumentationTree(
```

Comment:
Rename to `updateDocumentationTreeByPost` since it does the same as `updateDocumentationTree`.